convert-hf-to-gguf-update.py

```diff
@@ -106,12 +107,26 @@ def download_model(model):
     if tokt == TOKENIZER_TYPE.SPM:
         files.append("tokenizer.model")
 
-    for file in files:
-        save_path = f"models/tokenizers/{name}/{file}"
-        if os.path.isfile(save_path):
-            logger.info(f"{name}: File {save_path} already exists - skipping")
-            continue
-        download_file_with_auth(f"{repo}/resolve/main/{file}", token, save_path)
+    if os.path.isdir(repo):
+        # Если repo это путь на файловой системе, копируем директорию
```

Comment:
This comment should preferably be in English, I think :)

Reply:
Opps, sorry :)

Reply:
Fixed